go.mod

```diff
@@ -1,19 +1,53 @@
 module github.com/cert-manager/csi-lib
 
-go 1.16
+go 1.17
```

Comment:
Is the minimum version of go required to use this library actually changing?

Reply:
Ah, saw your other comment - it seems like it is then 😊 I'm guessing 'go build ./...' won't succeed without this patch + without 1.17?

Reply:
Yeah, I've created https://github.com/jetstack/testing/pull/621 to update the go version used for testing.